Libraries/LibGfx/ImageFormats/PNGLoader.cpp

```diff
@@ -133,6 +134,9 @@ ErrorOr<void> PNGImageDecoderPlugin::initialize()
     if (color_type == PNG_COLOR_TYPE_GRAY || color_type == PNG_COLOR_TYPE_GRAY_ALPHA)
         png_set_gray_to_rgb(png_ptr);
 
+    if (interlace_type == PNG_INTERLACE_ADAM7)
```

Comment:
A bit of a nitpick, but I think `interlace_type != PNG_INTERLACE_NONE` makes it clearer that we are calling this method for any interlaced image.

Reply:
Done.